federatedscope/cv/dataset/leaf_cv.py

```diff
@@ -91,6 +91,7 @@ def raw_file_names(self):
     def download(self):
         # Download to `self.raw_dir`.
         url = '/xxx/xxx/share'
+        url = "/mnt/gaodawei.gdw/data/"
```

Comment:
We'd better change the `url` to `cfg.data_path`. @rayrayraykk

Reply:
This is for local unit tests, I have modified it in the new version